core/src/main/java/com/linecorp/armeria/server/auth/AuthorizationStatus.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.auth;
+
+import javax.annotation.Nullable;
+
+import com.linecorp.armeria.common.annotation.UnstableApi;
+
+/**
+ * Provides a status of the request authorization operation, optionally combined with {@link AuthSuccessHandler}
+ * and {@link AuthFailureHandler} to facilitate custom status handling.
+ */
+@UnstableApi
+public interface AuthorizationStatus {
+
+    /**
+     * Creates {@link AuthorizationStatus} based on boolean status.
+     */
+    static AuthorizationStatus of(boolean status) {
```

Comment:
status -> authorized?

Reply:
Same comment as the above. No problem to make this change, but I would like to get the design approach validated first.